tests/test_mesh.py

```diff
@@ -546,7 +546,7 @@ def test_matlib():
     }
     m = gen_mesh(mats=mats)
     for i, ve in enumerate(mesh_iterate(m.mesh)):
-        assert_is(m.mats[i], mats[i])
+        assert_equal(m.mats[i], mats[i])
```

Comment:
thx for the fix, but I am not sure it is the intended behavior :(

Reply:
That's true. `assert_is` is more strict than `assert_equal`. I am not sure about this neither.

Reply:
thinking a little bit more, in this case the assert_is is not possible to maintain...
I'll merge your PR.

THX